pkg/state/transaction_checker_test.go

```diff
@@ -1281,22 +1283,11 @@ func TestCheckSetScriptWithProofsCheckScriptComplexity(t *testing.T) {
 				Estimation: MaxCallableScriptComplexityV6,
 				Verifier:   MaxVerifierScriptComplexityReduced + 1,
 			},
-			libVersions:               []int{1, 2},
+			libVersions:               []ast.LibraryVersion{ast.LibV1, ast.LibV2},
 			isDapp:                    false,
 			reducedVerifierComplexity: false,
 			valid:                     false,
 		},
-		// unknown lib version
-		{
-			estimationStub: ride.TreeEstimation{
-				Estimation: MaxCallableScriptComplexityV34,
-				Verifier:   MaxVerifierScriptComplexity,
-			},
-			libVersions:               []int{100500},
-			isDapp:                    true,
-			reducedVerifierComplexity: false,
-			valid:                     false,
-		},
```

Comment:
Why did you remove these lines?

Reply:
Where is no way to create 100500 libVersion now.

Reply:
I'll restore this test with different but incorrect version